cpp-17.0/index.cc

```diff
@@ -0,0 +1,24 @@
+/*
+  'req' variable has:
+    'headers' - object with request headers
+    'payload' - object with request body data
+    'env' - object with environment variables
+  'res' variable has:
+    'send(text, status)' - function to return text response. Status code defaults to 200
+    'json(obj, status)' - function to return JSON response. Status code defaults to 200
+  
+  If an error is thrown, a response with code 500 will be returned.
+*/
+
+#include <iostream>
+#include <string>
+
+static RuntimeResponse &main(const RuntimeRequest &req, RuntimeResponse &res) {
+
+    // Appwrite's SDK for C++ is currently under the works and shall be available soon
```

Comment:
I added this comment here keeping in mind the open PR for the C++ SDK in the SDK Generator Repo: https://github.com/appwrite/sdk-generator/pull/363